examples/regression_test1/main.cpp

```diff
@@ -59,7 +59,34 @@ void test_fields_and_constructor() {
   zngur_dbg(v4.f1.field2.len());
 }
 
+void test_field_underlying_conversions() {
+  auto scope = rust::crate::Scoped::new_("Test Field* underlying conversions"_rs);
+
+  rust::Tuple<int32_t, rust::std::string::String> pair{42, "hi"_rs.to_owned()};
+
+  // FieldOwned conversion to Ref and value
+  rust::Ref<int32_t> r0 = pair.f0;
+  int32_t v0 = pair.f0;
+  zngur_dbg(v0);
+  // TODO: Add support for conversion to T for all fields.
+  // rust::std::string::String v1 = pair.f1;
```

Comment:
@HKalbasi I'm a little stumped here. Still getting my bearings around cpp.rs, is there an obvious place to add an operator T for these?